src/get_games.py

```diff
@@ -144,5 +147,16 @@ def download_games(name: str,
 
 
 if __name__ == '__main__':  # pragma: no cover
+    auth = setup(path="./")
+
+    user_stats = lichess.api.user('carequinha')
+    time_creation = user_stats['createdAt']
+    time_last_seen = user_stats['seenAt']
+    inc = int(time_last_seen - time_creation / 100)
+    time_mock = time_creation + inc
+
+    # this is a mock to demo how to use this function, the timeframes are very small
     download_games('carequinha',
-                        is_rated=True)
+                   perf_type="blitz",
```

Comment:
Check for methods that use "pref_type" or "preftype"  and uniformise usage. What does this mean, by the way?

Reply:
Sorted. Typo in the docs. There is a game time format called blitz usually between 3 and 10 min games.